.github/workflows/actions.yml

```diff
@@ -23,9 +27,7 @@ jobs:
           ruby -v
           bundler -v
           java -version
-      - name: "Gradle: setup"
-        run: ./gradlew setup --no-daemon --stacktrace
-      - name: "Gradle: build site"
-        run: ./gradlew buildSite --no-daemon --stacktrace
-      - name: "Gradle: publish site"
-        run: ./gradlew pushSite --no-daemon --stacktrace
+      - name: Gradle setup, build and push
```

Comment:
yes we could use just gradlew, but the gradle command handles all kind of things internally, eg. using gradlew when provided and so on